addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/CrossSiteScriptingScanRuleUnitTest.java

```diff
@@ -687,6 +687,7 @@ protected Response serve(IHTTPSession session) {
         HttpMessage msg = this.getHttpMessage(test + "?name=test");
 
         this.rule.init(msg, this.parent);
+        this.rule.setAlertThreshold(AlertThreshold.HIGH);
```

Comment:
Should be in a new test.

Reply:
ok, will do

Reply:
hum, I can duplicate the test with the threshold High, but does it make sense? As is, we are basically saying the XSS is found even after raising the bar to High.
With two tests we have the advantage of ensuring the expected payload is used in both scenarios.
So just want to confirm the addition @thc202, before pushing.

Reply:
You can turn this into a parameterized test. But a test should be added that fails before your change (doesn't identify the vuln) but does after your changes.